Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCardViewModel.swift

```diff
@@ -92,6 +93,19 @@ final class PPOProjectCardViewModel: PPOProjectCardViewModelType {
 
   typealias Action = PPOProjectCardAction
   typealias Alert = PPOProjectCardAlert
+
+  // Add this static function at the end of the class
```

Comment:
Adding a comment; it's there because of the `actions` tuple, for some reason Xcode won't synthesize with it (even though it's doing so anyway for `lhs.actions == rhs.actions`).